### PR TITLE
LPS-144632 Replace Navigation Menu Tree View with new component

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/_NavigationMenuItemsTree.scss [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
 	.cadmin {
 		.navigation-menu-items-tree {
 			color: $cadmin-white;
 			font-size: 14px;
-			margin-left: 4px;
-			margin-top: 16px;
+			padding-left: 2rem;
+			padding-top: 1rem;
+
+			.component-expander {
```
Comment (victorg1991) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/_NavigationMenuItemsTree.scss:11]: can we use our own classes here? 😇

Comment (sandrodw3) [reply modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/_NavigationMenuItemsTree.scss:11]: We have agreed that we will open a ticket to FI so they can make the component more accesible. In the meanwhile, we are sending this.

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**